inc/Menu/Item/Revert.php

```diff
@@ -7,15 +7,18 @@
  *
  * Quick revert to the currently shown page revision
  */
-class Revert extends AbstractItem {
+class Revert extends AbstractItem
+{
 
     /** @inheritdoc */
-    public function __construct() {
+    public function __construct()
+    {
         global $REV;
         global $INFO;
+        global $INPUT;
         parent::__construct();
 
-        if(!$REV || !$INFO['writable']) {
+        if (!$REV || !$INFO['writable'] || $INPUT->server->str('REMOTE_USER') === '') {
```

Comment:
In `AbstractUserAction` I saw `!$INPUT->server->str('REMOTE_USER')`. Should we use strict equal or regular equal here?